### PR TITLE
Add list of J4 language packs and mark them as comptible

### PR DESCRIPTION
**Pull request by zero-24**
cc @Bakual lets go with that for now and add a to-do item for 3.10.2 whether we ship with the J4 list directly.

### CODE REVIEW

**Comment by wilsonge** [comment]
What's this for? We shouldn't be hosting these packs on joomlacode

File: translation-cli/translation_cron.php [hunk 1]
```diff
@@ -68,6 +68,55 @@ final class TranslationCron
 	 */
 	private $verbose = false;
 
+	/**
+	 * The list of J4 language packs to mark them as comptible
```
Comment (brianteeman) [inline translation-cli/translation_cron.php:72]:
```suggestion
	 * The list of J4 language packs to mark them as compatible
```

File: translation-cli/translation_cron.php [hunk 2]
```diff
@@ -68,6 +68,55 @@ final class TranslationCron
...
+	 *
+	 * @var array
+	 */
+	private $j4LanguagePack = array(
+		'ar-AA',
```
Comment (brianteeman) [inline translation-cli/translation_cron.php:77]: wouldn't this be better alpha sorted

Comment (zero-24) [reply translation-cli/translation_cron.php:77]: It is sorted as the downloads sites showed them to me.

**Comment by zero-24** [comment]
> What's this for? We shouldn't be hosting these packs on joomlacode

This makes sure the 3.x packages that have a 4.x package "claim" to be 4.x compatible, as the update will be shown just after the upgrade to J4.

File: translation-cli/translation_cron.php (hunk 2 above)
Comment (brianteeman) [reply translation-cli/translation_cron.php:77]: they are alpha sorted by real name.

think about it. if hebrew is added where will you put it in the list. 
if welsh is removed how will you find it in this list.

keeping the sort order based on the download site will mean you need to build the entire list whenever there is a change

Comment (zero-24) [reply translation-cli/translation_cron.php:77]: That is a good point, will resort it later.

File: translation-cli/translation_cron.php [hunk 3]
```diff
@@ -68,6 +68,55 @@ final class TranslationCron
...
+		'ru-RU',
+		'sk-SK',
+		'sl-SI',
+		'es-ES',
+		'sw-KE',
+		'sv-SE',
+		'ta-IN',
+		'th-TH',
+		'uk-UA',
+		'cy-GB',
```
Comment (brianteeman) [inline translation-cli/translation_cron.php:117]:
```suggestion
		'ar-AA',
		'ca-ES',
		'cy-GB',
		'da-DK',
		'de-AT',
		'de-CH',
		'de-DE',
		'de-LI',
		'de-LU',
		'el-GR',
		'en-AU',
		'en-US',
		'eo-XX',
		'es-ES',
		'eu-ES',
		'fa-IR',
		'fr-FR',
		'hr-HR',
		'hu-HU',
		'id-ID',
		'it-IT',
		'ja-JP',
		'ka-GE',
		'km-KH',
		'lv-LV',
		'nb-NO',
		'nl-BE',
		'nl-NL',
		'pl-PL',
		'pt-PT',
		'ro-RO',
		'ru-RU',
		'sk-SK',
		'sl-SI',
		'sv-SE',
		'sw-KE',
		'ta-IN',
		'th-TH',
		'uk-UA',
		'zh-CN',
		'zh-TW',
```

Comment (zero-24) [reply translation-cli/translation_cron.php]: Thanks 👍

**Comment by zero-24** [comment]
@wilsonge would you mind merging this here in?

**Pull request merged**